src/BayesFilters/include/BayesFilters/LTIStateModel.h

```diff
@@ -0,0 +1,44 @@
+#ifndef LTISTATEMODEL_H
+#define LTISTATEMODEL_H
+
+#include <Eigen/Dense>
+
+#include <BayesFilters/LinearStateModel.h>
+
+namespace bfl {
+    class LTIStateModel;
+}
+
+
+class bfl::LTIStateModel : public bfl::LinearStateModel
+{
+public:
+    LTIStateModel(Eigen::MatrixXf& transition_matrix,
+                  Eigen::MatrixXf& noise_covariance_matrix);
+
+    virtual ~LTIStateModel() noexcept { };
+
+    void propagate(const Eigen::Ref<const Eigen::MatrixXf>& cur_states,
+                           Eigen::Ref<Eigen::MatrixXf> prop_states) override;
```

Comment:
Put this line next to the previous one 😄

Reply:
Done!